frontend/src/pages/data.tsx

```diff
@@ -0,0 +1,94 @@
+export type head = {
+    id: keyof Data,
```

Comment:
was genau bedeutet das?

Reply:
"The keyof operator takes an object type and produces a string or numeric literal union of its keys." .. kanns auf deutsch nicht beschreiben. kommt halt mit der MUI Komponente..